drivers/bluetooth/hci/spi.c

```diff
@@ -201,22 +204,21 @@ static int configure_cs(void)
 		return -EIO;
 	}
 
-	gpio_pin_configure(cs_dev, GPIO_CS_PIN,
-			   GPIO_DIR_OUT | GPIO_PUD_PULL_UP);
-	gpio_pin_write(cs_dev, GPIO_CS_PIN, 1);
+	gpio_pin_configure(cs_dev, GPIO_CS_PIN, GPIO_OUTPUT | GPIO_CS_FLAGS);
+	gpio_pin_set(cs_dev, GPIO_CS_PIN, 1);
```

Comment:
This PR may need to be rebased but latest GPIO API supports `GPIO_OUTPUT_ACTIVE` flag. The two calls can be replaced with
```
	gpio_pin_configure(cs_dev, GPIO_CS_PIN, GPIO_OUTPUT_ACTIVE | GPIO_CS_FLAGS);
```